site/data/sidebar.yml

```diff
@@ -31,10 +31,10 @@
   pages:
     - title: Overview
       draft: true
+    - title: Lib for Orange
```

Comment:
TBH, I'm not a big fan of this wording. Isn't "Library" enough? Or what do you think about "Custom Libraries"?